workbox/workbox-cli-config.js

```diff
@@ -0,0 +1,14 @@
+const paths = require('../config/paths.json') // specify paths to main working directories
+
+module.exports = {
+  globDirectory: paths.public,
+  globPatterns: [
+    '**/*.{html,js,css,png}'
+  ],
+  swDest: paths.public + 'sw.js',
```

Comment:
Nit: I know it's common for 'sw' to be used in tutorials and such but we prefer to avoid abbreviations. Could anywhere there's SW could we use 'ServiceWorker' instead?